### PR TITLE
Redox / jeherve: updates

### PR DESCRIPTION
**Pull request by jeherve**
## Description

This PR brings in some updates to one of the keymaps for the Redox keyboard. It fixes a small issue I had when switching to the `NAV` layer, and brings in some changes and new macros. I also switched to using `send_unicode_hex_string` for one of my macros. 

Kudos to @drashna for his help [on Reddit](https://www.reddit.com/r/olkb/comments/a7om54/qmk_issues_with_tt_getting_stuck_on_lower_layer/)!

Details of the changes:

- 2 new Macro keys, to invoke [Alfred](https://www.alfredapp.com/) and [1Password](https://1password.com/).
- No more Right Command key or Expose key, I didn't use those much.
- Updates to use newer QMK function to call `¯\_(ツ)_/¯`
- Fix: you can now move back to the base layer when double tapping the thumb keys to switch to the Nav layer.

## Types of changes

- [ ] Core
- [ ] Bugfix
- [ ] New Feature
- [ ] Enhancement/Optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/Layout/Userspace (addition or update)
- [ ] Documentation

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document. (https://docs.qmk.fm/#/contributing)
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/redox/keymaps/jeherve/keymap.c [hunk 1]
```diff
@@ -25,31 +25,38 @@ enum custom_keycodes {
...
-				register_code (KC_RSFT); TAP_ONCE (KC_0); TAP_ONCE (KC_MINS); unregister_code (KC_RSFT);
-				TAP_ONCE (KC_SLSH);
-				unicode_input_start (); register_hex(0xaf); unicode_input_finish();
+				send_unicode_hex_string("00AF 005C 005F 0028 30C4 0029 005F 002F 00AF");
 			}
 			return false;
 			break;
+    case INVOKE_1P:
+      if (record->event.pressed) {
+        SEND_STRING(SS_DOWN(X_LGUI)SS_DOWN(X_LALT));
```
Comment (drashna) [inline keyboards/redox/keymaps/jeherve/keymap.c]: Could could make this all one line, actually. 

```c
SEND_STRING(SS_DOWN(X_LGUI)SS_DOWN(X_LALT)SS_TAP(X_BSLASH)SS_UP(X_LGUI)SS_UP(X_LALT));
```

File: keyboards/redox/keymaps/jeherve/keymap.c [hunk 2]
```diff
@@ -25,31 +25,38 @@ enum custom_keycodes {
 	WAVE,
 	YOSHI,
 	THUMB_UP,
-	NBSP
+	NBSP,
+  INVOKE_1P,
```
Comment (drashna) [inline keyboards/redox/keymaps/jeherve/keymap.c]: You may want to try to keep thins to 7 characters long.  That's the standard length, and is what will fit better "visually". 

This isn't a requirement, just a suggesion.

File: keyboards/redox/keymaps/jeherve/keymap.c (hunk 1 above)
Comment (jeherve) [reply keyboards/redox/keymaps/jeherve/keymap.c]: 👍 	Done in ef8a4787f7d46ad3efb63fa77e69e45ed372f59a

File: keyboards/redox/keymaps/jeherve/keymap.c (hunk 2 above)
Comment (jeherve) [reply keyboards/redox/keymaps/jeherve/keymap.c]: Makes sense. Done in 6bd230a1d19f976100a846e9d2c9fa7413531089

**Comment by jeherve** [comment]
> We recently merged a pretty big overhaul to the unicode functionality. So not only does it properly persist and load the input mode, as well as added keycodes to select the mode too.

That looks like it was quite a big change, and very recent too! Kudos, and thanks for the hard work.

I am trying to wrap my head around the changes, and I am sure how I should approach this. Right now my keyboard is always set to `UC_OSX`; I don't need to do anything to enable Unicode mode, I want it on all the time. 

If I switched to using the new keycode to select the mode, that means I would have to hit a key to switch modes and I am not sure that's ideal. That said, and since all my unicode use happens on the same layer, maybe I could switch to unicode mode every time I land on that layer, and switch back when I leave the layer.
What do you think? What would be the best approach here? Should I stick to what I use today?

**Comment by drashna** [comment]
Well, the docs are a good place to start. :)
But if you have the mode set already, you should be fine, already.

However, I'd recommend adding:
```c
void eeconfig_init_user(void) {
  set_unicode_input_mode(UC_OSX);
}
```

This way, if the eeprom gets reset, it should default to the correct input method.

**Comment by jeherve** [comment]
👍 	Done in a555ab7bc072aa1b1f80843c1906eb68d089a29e

**Comment by drashna** [comment]
Thanks!

**Pull request merged**